packages/react-instantsearch-core/src/connectors/connectRange.js

```diff
@@ -151,8 +150,8 @@ function refine(props, searchState, nextRefinement, currentRange, context) {
   const nextMinAsNumber = !isMinReset ? parseFloat(nextMin) : undefined;
   const nextMaxAsNumber = !isMaxReset ? parseFloat(nextMax) : undefined;
 
-  const isNextMinValid = isMinReset || _isFinite(nextMinAsNumber);
-  const isNextMaxValid = isMaxReset || _isFinite(nextMaxAsNumber);
+  const isNextMinValid = isMinReset || isFinite(nextMinAsNumber);
```

Comment:
This is not exactly the same as before since we don't check for the `number` type before, although it should work here.

Reply:
```
> isFinite(NaN)
< false
> isFinite(undefined)
< false

> Number.isFinite(NaN)
< false
> Number.isFinite(undefined)
< false
```

AFAICT the behaviour is the same for any value that can be passed in here, since the real danger of isFinite is passing it strings of numbers